gpslogger/src/main/java/com/mendhak/gpslogger/loggers/customurl/CustomUrlLogger.java

```diff
@@ -89,8 +89,7 @@ public void annotate(String description, Location loc) throws Exception {
                 PreferenceHelper.getInstance().getCurrentProfileName(),
                 Session.getInstance().getTotalTravelled());
 
-
-        manager.sendByHttp(finalUrl,httpMethod, finalBody, finalHeaders, basicAuthUsername, basicAuthPassword);
+        manager.sendByHttp(finalUrl, httpMethod, finalBody, finalHeaders, basicAuthUsername, basicAuthPassword);
```

Comment:
User preference check moved into Custom URL Manager

Reply:
Moved back, this was the right place to begin with.  Just slightly changed it to be passed in via the constructor.